src/borg/lrucache.py

```diff
@@ -28,6 +28,14 @@ def __delitem__(self, key):
     def __contains__(self, key):
         return key in self._cache
 
+    def get(self, key, default=None):
+        value = self._cache.get(key, default)
+        if value is default:
+            return value
```

Comment:
for detecting that something is not in _cache (and thus needs no further treatment in 35/36), the code needs to use some unique default value that can not exist as a legitimate value.

So, it can't be just None, but must be something like doesnotexistasvalue = object().